packages/xrpl-crypto/LICENSE

```diff
@@ -0,0 +1,15 @@
+ISC License
+
+Copyright (c) 2012-2015 Ripple Labs Inc.
```

Comment:
TODO: this seems quite old

Reply:
```suggestion
Copyright (c) 2012-2023 The XRPL developers
```

Reply:
Do you not have to list previous copyright holders? I vaguely recall prior situations where listing them all was necessary ?